server/middleware/auth.go

```diff
@@ -0,0 +1,33 @@
+package middleware
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/joshprzybyszewski/cribbage/auth"
+)
+
+func Auth() gin.HandlerFunc {
```

Comment:
Do you have a link to the example you followed for this?

Reply:
I think [this](https://sosedoff.com/2014/12/21/gin-middleware.html) was it